jobs/process_unassigned_chunks.go

```diff
@@ -283,6 +283,7 @@ func InsertTreasureChunks(chunks []models.DataMap, treasureChunks []models.DataM
 			treasureChunksInserted++
 			i++ // skip an iteration
 		} else if _, ok := treasureChunksMapped[chunks[i].ChunkIdx+idxTarget]; ok {
+			// LOOK INTO THIS
```

Comment:
?

Reply:
Oops.  This code looked suspicious to me so I made a note to take a closer look at it but forgot to remove the note.  Will remove it now.